app/workers/subject_workflow_status_count_worker.rb

```diff
@@ -0,0 +1,21 @@
+class SubjectWorkflowStatusCountWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :really_high
+
+  sidekiq_options congestion: {
+    interval: ENV.fetch("sws_count_worker_interval", 30),
```

Comment:
I just thought about that this afternoon. I'd be cool to integrate configuration values like this with Flipper, and have a live admin panel for them. In general we should probably be putting more non-critical things under flags anyways, so we can toggle them off under load.